entity-view/testsuite/src/test/java/com/blazebit/persistence/view/testsuite/update/converter/EntityViewUpdateBlobTest.java

```diff
@@ -215,7 +216,7 @@ public void testSetBlob() throws Exception {
     // Oracle keeps LOBs open/tied to a result set so we can't write to it by means of setBytes.
     // For Oracle it would be more appropriate to treat writes like "replacements" i.e. remember the written bytes and replace the underlying object on flush
     // NOTE: No Datanucleus support yet
-    @Category({ NoDatanucleus.class, NoEclipselink.class, NoOracle.class })
+    @Category({ NoDatanucleus.class, NoEclipselink.class, NoOracle.class, NoH2.class})
```

Comment:
Again, please add a note explaining why this test fails on H2

Reply:
Yes. There reason is that it broke in a new H2 :( still investigating that, and report issues where needed